hoth_api.yaml

```diff
@@ -445,39 +447,45 @@ components:
       type: object
       properties:
         period:
-          description: "The period of performance for a specific period sheet."
+          description: "A base or option period."
```

Comment:
Not sure if we want to remove the `" "` for the `description` like you did for the other ones.

Reply:
We may need to do that for the entire file someday.  There's definitely a mix.